### DISCUSSION TITLE
geometric compatibility factor or M prime by schmid factor

### DISCUSSION THREAD

**Discussion by mugikumar** [mtex-toolbox/mtex#2109] [Ask Anything]
Hi everyone,
 I was trying to construct the map of geometric compatibility factor or m prime in the misorientation space with help of Schmid factor but it was showing error in Matlab, but when i was using the demo set it was working fine. i think slip system for bcc have some issue for array operation. please somebody help me to resolve it and find the attachment
if i have a workspace of m prime and misorientation of two grains. is this possible to construct the map of misorientation vs m prime. for example image is attached below
![Screenshot 2024-04-04 022629](https://github.com/mtex-toolbox/mtex/assets/109027461/815cc86b-5c4b-4cd4-a128-35eac22c406d)
![mori of m prime ](https://github.com/mtex-toolbox/mtex/assets/109027461/a22cd108-49a3-46fe-ab85-6c69e254523b)

**Comment by Tijmenvermeij** [comment]
I think you're going to need to post some more code to see what is going on before anyone can help you.
